tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -23,6 +23,12 @@ import (
 // by this logger will have an identifier containing `aws-ec2` tag.
 var log = logger.Log.WithName("aws-ec2")
 
+// Constant for port number
+const (
+	RDP_PORT   = 3389
```

Comment:
I clearly stated that this should be done in a separate commit or PR. https://github.com/openshift/windows-machine-config-operator/pull/32/commits/54a92deb72b38e77de019572b3f4d81b3a84fead#r328349575

I will let it go this time but ignoring PR feedback is not acceptable in the future.

Reply:
Sorry about that. I will be more careful next time.